src/components/instructor/InstructorSectionsView.js

```diff
@@ -12,14 +15,72 @@ import React, {useState, useEffect} from 'react';
 
 const InstructorSectionsView = (props) => {
 
-    
+    const headers = ['SecNo', 'CourseId', 'SecId',  'Year', 'Semester', 'Building', 'Room', 'Times', '', ''];
+
+    const [sections, setSections] = useState([]);
+    const [message, setMessage] = useState('');
+
+    const location = useLocation();
+    const term = location.state;
+
+    const instructorEmail = 'dwisneski@csumb.edu';
+
+    const fetchSections = async () => {
+        if (term.year === '' || term.semester === '') {
```

Comment:
`term.year` should also be checked if it is a number, otherwise the database throws back a complicated error:
```javascript
isNaN(Number(term.year))
```